apis/react-suspense.md

```diff
@@ -16,7 +16,7 @@ React Suspense is a generic way for components to suspend rendering while they l
   - showing a **fallback** UI if the duration of the suspense exceeds a threshold
   - **resuming** render when the fetch is fulfilled and the cache read obtains a value
 - Render is only complete and committed to the DOM when either:
-  - `maxDuration` is exceeded and the **fallback** UI is shown. React ships with built in maxDuration expiry limits (up to 5 seconds, iirc), and will use the smaller of the builtin limit or of whatever you specify (see below).
+  - a hueristic and different mechanism decides that a **fallback** UI is shown. React ships with [hard-coded built-in expiry limit of 150ms](https://github.com/facebook/react/pull/15272#issue-265972492).
```

Comment:
Not true anymore, https://github.com/facebook/react/pull/15367

Reply:
Thank you, @arianon.
You are right (and "fresh" with the information).

`150ms` is deleted - https://github.com/facebook/react/pull/15367/files#diff-a409dc1b2c8ece1cc1fa28fe42b481ceL1829
and uses a hueristic (Just Noticeable Difference (JND) boundary) only to compute the timeout duration.

I will update the documentation and update the PR.

Reply:
Thanks @arianon. I've updated the documentation & the PR.